src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java

```diff
@@ -404,13 +403,12 @@ private void trackVideo(String eventName, TrackPayload track) {
 
     switch (eventName) {
       case "Video Playback Started":
```

Comment:
Some of these cases are pretty big. These should be refactored into their own functions (e.g. this should be `VideoPlaybackStarted`.